calico-policies/private-network-isolation/au-syd/allow-private-services-pods.yaml

```diff
@@ -51,6 +51,11 @@ spec:
       - 166.9.52.5/32
       - 166.9.56.3/32
       - 166.9.56.20/32
+      # CIDR Blocks added on 7/26/2023
```

Comment:
How will customer cluster workers be able to send traffic to LogDNA via 10.x.x.x destination IPs?  I thought the only approved way to send traffic to services like LogDNA over the private network is to use CSEs?

Reply:
@bradbehle Thease are the CIDR blocks for private networks that we advertise. I am not sure how the calico-policies use these as I am not an expert in this area. We have been asked to update these policies for the sake of other services so I would like to know what these policies are used for. IF you think that the 10.x IPs are not needed I can remove them.

Reply:
I don't think those 10.x IPs are needed.  These policies are meant to be examples of what can be applied to IKS and ROKS cluster worker to restrict what connections can be made to/from those workers (mostly restricting egress traffic from the workers.  So the parts of this policy that you are updating are restricting what LogDNA services (IPs/CIDRs) the IKS/ROKS cluster workers can connect to.

So as an example, I assume something that IKS/ROKS cluster workers that are using LogDNA need to do is to connect out to LogDNA servers to export their logs to LogDNA?  In that case, whatever IPs/CIDRs the cluster workers might connect to in order to send their logs would be added to these policies.